Library/Formula/i2p.rb

```diff
@@ -0,0 +1,25 @@
+class I2p < Formula
+  desc "I2P is an anonymous overlay network - a network within a network"
+  homepage "https://geti2p.net"
+  url "https://download.i2p2.de/releases/0.9.21/i2pinstall_0.9.21.jar"
+  sha256 "0238ffc6ea44099ef4fe6c20913cb4eec675c2760aea07dfe7d499addcc89cf2"
+
+  depends_on :java => "1.6+"
+
+  def install
+    open("i2p_template.conf", "w") do |f|
+      f.write("INSTALL_PATH=#{prefix}")
+    end
+
+    system "java -jar i2pinstall_0.9.21.jar -options i2p_template.conf"
+    system "ln -s #{prefix}/i2psvc-macosx-universal-64 #{prefix}/i2psvc"
+
+    system "mkdir -p #{prefix}/bin"
+    system "cp #{prefix}/eepget #{prefix}/bin/"
+    system "cp #{prefix}/i2prouter #{prefix}/bin/"
```

Comment:
You can combine these three lines into `bin.install libexec/"eepget", libexec/"i2prouter"` or even:

```ruby
bin.write_exec_script Dir["#{libexec}/*"].select { |f| File.executable? f }
```